.changeset/add-ut-for-mfd-bytes-2024-1-8-3-25-28.md

```diff
@@ -0,0 +1,5 @@
+---
+"@azure-tools/typespec-client-generator-core": patch
+---
+
+add-ut-for-mfd-bytes
```

Comment:
Can you describe what the change is here

Reply:
if this is just test change the type to `none` and you can remove the description